tests/end_to_end/features/steps/spark_registeration.py

```diff
@@ -136,17 +136,31 @@ def step_impl(context):
     context.exception = None
 
 
-@when("I register a transformation")
-def step_impl(context):
-    @context.spark.df_transformation(
-        name="transactions_transformation",
-        inputs=[context.file],
-    )
-    def ice_cream_transformation(df):
-        """Unedited transactions"""
-        return df
+@when(
+    'I register a "{transformation_type}" transformation named "{name}" from "{sources}"'
+)
+def step_impl(context, transformation_type, name, sources):
+    source_list = sources.split(",")
+    if transformation_type == "DF":
+
+        @context.spark.df_transformation(
+            name=name,
+            inputs=source_list,
+        )
+        def some_transformation(df):
```

Comment:
I'm not seeing this, so apologies if this is obvious, but will this be tested with more than one input?

Reply:
For these tests, it looks like single inputs. You can see them in the [transformations.feature](https://github.com/featureform/featureform/blob/563dce3107ef9e39760faf317afc9d1e49809bb1/tests/end_to_end/features/transformations.feature#L11-L12) file